qcodes/dataset/data_set.py

```diff
@@ -342,6 +346,10 @@ def run_timestamp_raw(self) -> float:
     def description(self) -> RunDescriber:
         return self._description
 
+    @property
+    def metadata(self) -> Dict:
```

Comment:
should metadata be added to the persistent traits list?

Reply:
Yes, but the persistent traits do not exist yet, only on another feature branch.